### DISCUSSION TITLE
Menu | How to turn autofocus off

### DISCUSSION THREAD

**Discussion by Ruslan-I7** [mantinedev/mantine#3474] [Q&A]
Hello guys, 

How to turn autofocus off when u open a menu? I noticed that it is at least in version 5.2.4. But it is not in 5.10.2. Anyway I'd like to know how to manage it.
And yet, when u click outside `returnFocus` doesn't work. But if u use `closeOnEscape` then `returnFocus` works

**Comment by rtivital** [comment]
> How to turn autofocus off when u open a menu?

Menu does not have autofocus. It is not clear what you want to turn off.

> And yet, when u click outside returnFocus doesn't work. But if u use closeOnEscape then returnFocus works

It works as expected. If you close Menu by clicking outside, then the element that you've clicked on will be focused. If you close Menu with escape key, then focus will be returned to the last active element.

**Comment by Ruslan-I7** [comment]
> Menu does not have autofocus. It is not clear what you want to turn off.

Take a look [here](https://codesandbox.io/s/react-mantine-autofocus-menu-vmb23q?file=/src/App.js&resolutionWidth=410&resolutionHeight=224), please

Make the height of a screen less than the height of an open menu and open it,  u'll notice that u move to ur menu, from the header to the menu. It's similar to autofocus even if it's not. What if I don't want to jump like this? Do I need to add styles or there is some prop that works this case out? 

>  If you close Menu by clicking outside, then the element that you've clicked on will be focused.

A user can click just on the empty background where there are no any elements. And then he'll stay where he is. It will look like a user was moved not wishing for this. And if he wants to click on a menu again, he will have to scroll up again.

By the way, I added one more block and distance between them, now when I close by `ESC` I stay where I was moved. I think this thing may not please a user if it happens to him of course)) because it's the special case

**Comment by Ruslan-I7** [comment]
But anyway, I wonder if u could tell me how to not let move and stay where u clicked?
